### PR TITLE
Feature/caes 1063 implement permissions for team

### PR DESCRIPTION
**Pull request by cu4xxi**

### CODE REVIEW

File: packages/common/theme.js [hunk 1]
```diff
@@ -33,12 +33,12 @@ const theme = {
...
-    overlay: 13,
-    dropdown: 14,
-    modal: 15,
-    notification: 16,
+    basic: 101,
+    upBasic: 102,
+    overlay: 103,
+    dropdown: 104,
+    modal: 105,
+    notification: 106,
```
Comment (rtatarinov4xxi) [inline packages/common/theme.js:41]: It's big values for zIndex. Maybe it's redundant

File: packages/components/Item/Item.js [hunk 2]
```diff
@@ -89,218 +40,85 @@ const ItemComponent = ({
...
+        team_move_item: !!item._links?.team_move_item,
+        team_batch_share_item: !!item._links?.team_batch_share_item,
+        team_delete_item: !!item._links?.team_delete_item,
+      }
+    : {
+        __typename: PERMISSION_ENTITY.ITEM,
+        edit_item: !!item._links?.edit_item,
+        move_item: !!item._links?.move_item,
+        batch_share_item: !!item._links?.batch_share_item,
+        delete_item: !!item._links?.delete_item,
```
Comment (rtatarinov4xxi) [inline packages/components/Item/Item.js:85]: Why do you use snake_case here?

File: packages/common/theme.js (hunk 1 above)
Comment (cu4xxi) [reply packages/common/theme.js]: yep. fixed

File: packages/components/Item/Item.js (hunk 2 above)
Comment (cu4xxi) [reply packages/components/Item/Item.js:85]: this is a backend stuff. I decided to use snake_case vars on front end for identity.

**Comment by cu4xxi** [comment]
> This pull request **introduces 1 alert** and **fixes 11** when merging [eada3e0](https://github.com/caesar-team/caesar-web-client/commit/eada3e0adfd752a72f61678dbb615623522ee641) into [71fe0ff](https://github.com/caesar-team/caesar-web-client/commit/71fe0ff6b57d385d7883d5463871a1e3ba18d96f) - [view on LGTM.com](https://lgtm.com/projects/g/caesar-team/caesar-web-client/rev/pr-eefc24888cefcb7d22aa4dc440601bed85df1f27)
> 
> **new alerts:**
> 
> * 1 for Unused variable, import, function or class
> 
> **fixed alerts:**
> 
> * 9 for Unused variable, import, function or class
> * 2 for Duplicate character in character class

@beshkenadze I'm goimg to use it in the next implementation of permissions (when back end does this stuff)

**Pull request merged**